Makefile.scan-build

```diff
@@ -70,7 +70,7 @@ scan-build-analyze: clean
 	$(AD)mkdir -p '$(SCANBUILD_OUTPUTDIR)'
 	$(AD)env -i $(ENVVARS) \
 	    scan-build -o '$(SCANBUILD_OUTPUTDIR)' $(SCANBUILD_ARGS) \
-	      make -C $(CURDIR) all $(strip $(CMDVARS));
+	      make -C $(CURDIR) all $(strip $(CMDVARS)) FORCE_ASSERTS=1;
```

Comment:
But we don't want NDEBUG ("no debug"). An extra flag to disable defining NDEBUG was my first idea, but I don't know if it is the best solution.